src/index.css

```diff
@@ -1,27 +1,9 @@
-body {
```

Comment:
- [ ] It would be better if we align the user profile avatar to the booklist component as the Booklist title is aligned to the beginning of the booklist component